src/crimson/os/seastore/segment_cleaner.cc

```diff
@@ -143,6 +143,23 @@ void SpaceTrackerDetailed::dump_usage(segment_id_t id) const
   segment_usage[id].dump_usage(block_size);
 }
 
+SegmentCleaner::SegmentCleaner(config_t config, bool detailed)
+  : detailed(detailed),
+    config(config),
+    gc_process(*this)
+{
+  register_metrics();
+}
+
+void SegmentCleaner::register_metrics()
+{
+  namespace sm = seastar::metrics;
+  metrics.add_group("sc", {
```

Comment:
segment_cleaner, I think.  Eventually, these will be exposed via prometheus, etc, so we probably want to be verbose with naming so that users will have more clues as to what they are seeing.

Reply:
renamed to "segment_cleaner", but it's still not obvious that it's related to crimson or seastore, we can revise it later.

Reply:
Mmm, yeah. Perhaps some form of crimson.seastore. prefixing scheme? Something for later.